ocs_ci/ocs/ocp.py

```diff
@@ -475,6 +475,21 @@ def delete_project(self, project_name):
             return True
         raise CommandFailed(f"{project_name} was not deleted")
 
+    def clean_all_test_projects(self, project_name="test"):
+        all_ns = self.exec_oc_cmd(
```

Comment:
Why you need to define the same function again here? Why you just not calling the function from helpers? And if you really need to have defined this as method in the class, you should from this method call the function you have defined and just providing the object you would like to modify/delete as param.

Reply:
my bad - forgot to delete it after moved to helpers